File: 344.Reverse String/Solution.cpp
#include <iostream>
#include <string>

using namespace std;

class Solution {
    
    public:
        string reverseString(string s) {
            int len = (int)s.size();
            for (int i = 0; i < len / 2; i++) {
                char ch = s[i];
                s[i] = s[len - i - 1];
                s[len - i - 1] = ch;
            }
            return s;
        }
};

File: 344.Reverse String/Solution.java
public class Solution {

    public String reverseString(String s) {
        char[] ch = s.toCharArray();
        for (int i = 0; i < ch.length / 2; i++) {
            char tmp = ch[i];
            ch[i] = ch[ch.length - i - 1];
            ch[ch.length - i - 1] = tmp;
        }
        return new String(ch);
    }
}
        
